README.md

```diff
@@ -1,2 +1,175 @@
-WP-DS-NPR-API
-==============
+# WordPress NPR Story API
+
+A collection of tools for reusing content from NPR.org supplied by Digital Services.
+
+Contributors: David Moore, Jeremy Dunn, kgrantNPR, Kevin Moylan, Ryan Nagle, Adam Schweigert, jackbrighton
+
+Requires at least: 3.8.14
+
+Tested up to: 4.5.2
+
+Stable tag: 1.5.2
+
+License: GPLv2
+
+License URI: http://www.gnu.org/licenses/gpl-2.0.html
+
+== Description ==
+
+WordPress NPR Story API Plugin provides push and pull functionality with the NPR API along with a user-friendly administrative interface.
+
+NPR's API is a content API, which essentially provides a structured way for other computer applications to get NPR stories in a predictable, flexible and powerful way. The content that is available includes audio from most NPR programs dating back to 1995 as well as text, images and other web-only content from NPR and NPR member stations. This archive consists of over 250,000 stories that are grouped into more than 5,000 different aggregations.
+
+Access to the NPR API requires an API Key which you can get by [registering for an NPR account](http://www.npr.org/account/signup).
+
+The WordPress plugin is being developed as an Open Source plugin by NPR Digital Services. If you would like to suggest features or bug fixes, or better yet if you would like to contribute new features or bug fixes please contact Digital Services through our support page at [http://info.ds.npr.org/support.html](http://info.ds.npr.org/support.html).
```

Comment:
it's ok to mention this here, but the repo should also have a contributing.md file at the root level that covers how to contribute in more detail